src/backend/joanie/tests/core/test_api_admin_organizations.py

```diff
@@ -345,3 +345,23 @@ def test_admin_api_organization_delete(self):
         )
 
         self.assertEqual(response.status_code, 204)
+
+    def test_admin_api_organization_bulk_delete(self):
+        """
+        Staff user should be able to delete multiple organizations.
+        """
+        admin = factories.UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=admin.username, password="password")
+        organizations = factories.OrganizationFactory.create_batch(3)
+        factories.OrganizationFactory()
+        data = {"id": [str(organization.id) for organization in organizations]}
+        response = self.client.delete(
+            "/api/v1.0/admin/organizations/",
+            data=data,
+            content_type="application/json",
+        )
+        content = response.json()
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(models.Organization.objects.count(), 1)
+        self.assertEqual(len(content["deleted"]), 3)
+        self.assertFalse("error" in content)
```

Comment:
Ditto, maybe we should add a test method to complete it : `test_admin_api_organization_bulk_delete_error` here as well ?

Reply:
There is no relation to an organization that is protected